### PR TITLE
Keymanager 'disk' implementation

### PR DESCRIPTION
**Pull request by maxlambrecht**
**Pull request check list**

- [X] Proper tests/regressions included?
- [ ] Documentation updated?

**Affected functionality**
Add a new `KeyManager` option for managing Keys in the Galadriel Server. This options adds the possibility of restarting the Galadriel Server without affecting the onboarded Harvesters, as the Server will be able to continue validating the issued JWT tokens.

**Description of change**
Implementation of a `KeyManager` that stores keys in disk.

**Which issue this pull requests fixes**

### CODE REVIEW

File: pkg/common/keymanager/disk.go [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+			PrivateKey: signer,
+			PublicKey:  signer.Public(),
+			id:         id,
+		}
+	}
+
+	return nil
+}
+
+// SaveKeysToDisk saves the keys in the key manager to disk.
```
Comment (mgbcaio) [inline pkg/common/keymanager/disk.go:92]:
```suggestion
// saveKeysToDisk saves the keys in the key manager to disk.
```

File: pkg/common/keymanager/keymanager.go [hunk 2]
```diff
@@ -8,7 +8,7 @@ import (
 	"github.com/HewlettPackard/galadriel/pkg/common/cryptoutil"
 )
 
-// KeyManager provides a common interface for managing keys.
+// Memory provides a common interface for managing keys.
 type KeyManager interface {
```
Comment (mgbcaio) [inline pkg/common/keymanager/keymanager.go:12]: Change interface name to match the new comment ?

**Review by mgbcaio — COMMENTED** [review]
just 2 comments, for now. looks good to me.

File: pkg/common/keymanager/keymanager.go (hunk 2 above)
Comment (maxlambrecht) [reply pkg/common/keymanager/keymanager.go]: No, the comment is wrong.

**Review by mgbcaio — APPROVED** [review]

**Pull request merged**